media/js/privacy/data-preferences-init.es6.js

```diff
@@ -4,67 +4,38 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
-const preferenceCookieID = 'moz-1st-party-data-opt-out';
+import DataPreferencesCookie from './data-preferences-cookie.es6';
+
 const prefStatus = document.querySelector('.data-preference-status');
 const statusText = prefStatus.querySelector('.data-preference-text');
+const optOutButton = document.querySelector('.js-opt-out-button');
+const optInButton = document.querySelector('.js-opt-in-button');
 let optOutText;
 let optInText;
 
 function updateStatus() {
-    if (hasOptedOut()) {
+    if (DataPreferencesCookie.hasOptedOut()) {
         statusText.innerText = optOutText;
         prefStatus.classList.add('is-opted-out');
         prefStatus.classList.remove('is-opted-in');
+        optOutButton.disabled = true;
```

Comment:
I like this addition a lot. Noticed it more than I expected I would, in a good way.